conans/test/functional/toolchains/cmake/test_cmaketoolchain_paths.py

```diff
@@ -175,7 +175,10 @@ def package_info(self):
 @pytest.mark.parametrize(
     "find_root_path_modes", [find_root_path_modes_default, find_root_path_modes_cross_build],
 )
-def test_cmaketoolchain_path_find_package_real_config(settings, find_root_path_modes):
+@pytest.mark.parametrize(
+    "builddir", ["os.path.join('hello', 'cmake')", "self.package_folder"],
```

Comment:
```suggestion
    "builddir", ["os.path.join('hello', 'cmake')", "self.package_folder", '"."'],
```

Let's add also the relative "." case.